keyboards/keebformom/readme.md

```diff
@@ -0,0 +1,27 @@
+# Keeb For Mom
+
+![KeebForMom](https://user-images.githubusercontent.com/82454371/181726757-91a95865-3763-4eb9-a22a-bf9bcdbccc05.JPG)
```

Comment:
Please rehost this on imgur.

Reply:
imgur has been  blocked in my country, can i use other image hosting ?

Reply:
i used imageshack as image hosting